src/api/Middleware/flightController.js

```diff
@@ -1,28 +1,82 @@
-import actionTypes from '../Actions/actionTypes';
+import { connectFlightController, disconnectFlightController, sendFlightControl } from '../Actions';
+import { actionTypes } from '../Actions/actionTypes';
```

Comment:
```suggestion
import actionTypes from '../Actions/actionTypes';
```
This currently does not work